packages/components/src/placeholder/style.scss

```diff
@@ -58,7 +58,6 @@
 	flex-direction: row;
 	width: 100%;
 	flex-wrap: wrap;
-	z-index: z-index(".components-placeholder__fieldset");
```

Comment:
Do you know why this was added in the first place?

Reply:
There's a comment on the other file: `// Should have higher index than the inset/underlay used for dragging` . Something may have changed since it was added, because the fieldsets are all usable without the z-index.  I also tested the dragging functionality (for image blocks and the like) and everything seems fine.